### PR TITLE
29 standardise location data (rebased)

### PR DESCRIPTION
**Pull request by michaelwood**
_See #39_

### CODE REVIEW

File: datastore/tests/test_additional_data_geolookups.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            #     "areaname": "Hartlepool 007B",
+            #     "areatype": "lsoa",
+            #     "msoa11cd": "E02002489",
+            #     "msoa11nm": "Hartlepool 007",
+            #     "ttwa11cd": "E30000215",
+            #     "ttwa11nm": "Hartlepool",
+            #     "msoa11hclnm": "Old Town & Grange",
+            #     "latitude": 54.68510808262816,
+            #     "longitude": -1.223715283044111,
+            # }
```
Comment (BibianaC) [inline datastore/tests/test_additional_data_geolookups.py:78]: @drkane are these uncommented lines to show the key, value that the test is checking or where left here by mistake?

Comment (drkane) [reply datastore/tests/test_additional_data_geolookups.py:78]: They were there to show me what the test was expecting to see while I was writing the text. Can probably be removed, but could be useful to see what the object is expected to look like?

**Comment by michaelwood** [comment]
I've just finished a run of all the current data:

```
real    6m19.740s
user    4m25.897s
sys     0m11.289s
```
Looking much more healthy! 

Before it is merged and deployed I am just going to run a sanity check against loading the output package for Grantnav so that we don't get any surprises.

File: datastore/tests/test_additional_data_geolookups.py (hunk 1 above)
Comment (BibianaC) [reply datastore/tests/test_additional_data_geolookups.py:78]: Yes, it can be useful too.

File: datastore/additional_data/sources/local_files.py [hunk 2]
```diff
@@ -10,6 +10,16 @@
 class LocalFilesSource(object):
     """ Adds additional data to grant data """
 
+    # Fields added to additional_data by this source.
+    # These are currently used in GrantNav.
+    ADDITIONAL_FIELDS = [
+        "recipientWardNameGeoCode",
+        "recipientWardName" "recipientDistrictName" "recipientDistrictGeoCode",
```
Comment (drkane) [inline datastore/additional_data/sources/local_files.py:17]: @michaelwood commas missing here - will that cause an issue?

**Pull request merged**